client-ts/package.json

```diff
@@ -23,11 +23,13 @@
   },
   "homepage": "https://github.com/aspnet/SignalR#readme",
   "devDependencies": {
+    "@types/node": "^8.0.19",
     "browserify": "^13.1.1",
     "del": "^2.2.2",
     "gulp": "^3.9.1",
     "gulp-typescript": "^3.1.3",
     "jasmine": "^2.5.2",
+    "msgpack-lite": "^0.1.26",
```

Comment:
Actually looking if we can use https://github.com/mcollina/msgpack5 which is not 0.x